Add druid logs (#4561)
DataDog/integrations-core: druid/README.md

Use the default configuration of your `druid.d/conf.yaml` file to activate the collection of your Druid service checks. See the sample [druid.d/conf.yaml][3] for all available configuration options.

#### Log collection

**Available for Agent >6.0**

> 1. Collecting logs is disabled by default in the Datadog Agent, enable it in the `datadog.yaml` file with:

Reviewer: ```suggestion
1. Collecting logs is disabled by default in the Datadog Agent, enable it in your datadog.yaml file:
```

    ```yaml
      logs_enabled: true
    ```

> 2. Add this configuration block to your `druid.d/conf.yaml` file to start collecting your Druid logs:

Reviewer: ```suggestion
2. Uncomment and edit this configuration block at the bottom of your `druid.d/conf.yaml`:
```
